### PR TITLE
Manage a mail list for hackers with travel reimbursement

### PR DESCRIPTION
**Pull request by ofpau**
Solves https://github.com/hackupc/mercuryhack/issues/39.

### CODE REVIEW

File: register/models.py [hunk 1]
```diff
@@ -104,7 +106,9 @@ def cancel(self):
             self.status = APP_CANCELLED
             self.save()
             m = MailListManager()
-            m.remove_recipient_from_list(self.sendgrid_id, m.WINTER_17_LIST_ID)
+            m.remove_recipient_from_list(self.sendgrid_id, m.W17_GENERAL_LIST_ID)
+            if self.scholarship:
+                m.remove_recipient_from_list(self.sendgrid_id, m.W17_TRAVEL_REIMB_LIST_ID)
```
Comment (casassg) [inline register/models.py:148]: if user is not in and we try to remove him. Is it an error?

**Review by casassg — APPROVED** [review]
lgtm, just a doubt I had

File: register/models.py (hunk 1 above)
Comment (ofpau) [reply register/models.py:148]: No error, if the user didn't belong to the list then removing her makes no change to sendgrid's side, but it doesn't cause any error. Just tested it (manually).

**Comment by ofpau** [comment]
It's been decided to manage the reimbursement mail list differently so this is no longer needed. See https://github.com/hackupc/mercuryhack/issues/51.

**Pull request closed without merging**